packages/apps-config/src/api/spec/plasm-parachain.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'MultiAddress',
+  LookupSource: 'MultiAddress',
+  ValidationDataType: {
+    validation_data: 'ValidationData',
+    relay_chain_state: 'Vec<Bytes>'
+  },
+  PersistedValidationData: {
+    parent_head: 'Bytes',
+    block_number: 'u32',
+    relay_storage_root: 'Hash',
```

Comment:
This field in missing in the API.

Reply:
This field still available in `rococo-v1` Polkadot primitives: https://github.com/paritytech/polkadot/blob/rococo-v1/primitives/src/v1.rs#L366

Reply:
Yes, I meant that the API definition didn't have that field.